docker/cortx-deploy/docker-compose.yml

```diff
@@ -26,7 +26,8 @@ services:
       context: ./
       dockerfile: ./Dockerfile
       args:
-       CENTOS_RELEASE: 7.9.2009
+       OS_TYPE: centos
```

Comment:
Do we need to add here an `OS` arg as well in case someone create image directly from `docker-compose.yml` and not `build.sh`?

Reply:
`OS_TYPE` and `OS_RELEASE` together create `OS` variable . This is done due to two reasons 

- For Rocky Linux container base image name is `rockylinux/rockylinux`
- If we need to move from Rocky Linux 8.4 to Rocky Linux 8.5 there will be only variable change and not the new service addition

Reply:
Added OS argument in docker-compose.yml. Developer can generate image using `docker-compose`
```
export TAG=2.0.0-0
docker-compose -f ./docker-compose.yml build --force-rm --compress
```